examples/todo-mvc-react/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+  "compilerOptions": {
+    "target": "ESNext",
+    "lib": ["DOM", "DOM.Iterable", "ESNext"],
+    "allowJs": false,
+    "skipLibCheck": true,
```

Comment:
A fresh project generated with `create-vite` has `skipLibCheck: false`, and it worked when outside of the `xstate` repo.
However there were type errors when running `pnpm run build` inside this repo, so I had to change `skipLibCheck` to `true`.